Makefile

```diff
@@ -454,25 +458,9 @@ clean-onprem:
 ############
 # Operator #
 ############
-
-# Current Operator version
-OPERATOR_VERSION ?= 0.0.3
-BUNDLE_OUTPUT_DIR ?= deploy/olm-catalog
-BUNDLE_METADATA_OPTS ?= --channels=alpha,ocm-2.3 --default-channel=alpha
-
-.PHONY: operator-bundle
-operator-bundle:
-	operator-sdk generate kustomize manifests --apis-dir internal/controller/api -q
-	# TODO(djzager) use this line to pin images in the future
-	# cd config/manager && kustomize edit set image controller=$(SERVICE)
-	kustomize build config/manifests | operator-sdk generate bundle -q --overwrite --version $(OPERATOR_VERSION) --output-dir $(BUNDLE_OUTPUT_DIR) $(BUNDLE_METADATA_OPTS)
-	# TODO(djzager) structure config/rbac in such a way to avoid need for this
-	rm $(BUNDLE_OUTPUT_DIR)/manifests/assisted-service_v1_serviceaccount.yaml
-	operator-sdk bundle validate $(BUNDLE_OUTPUT_DIR)
-
 # Build the bundle and index images.
 .PHONY: operator-bundle-build operator-bundle-update
-operator-bundle-build: operator-bundle
+operator-bundle-build: generate-bundle
```

Comment:
The failure we have in the ci-index is because it is trying to build the `operator-bundle` target. I like the rename to `generate-bundle` to include it in the `generate all`. But will probably have make `operator-bundle` to point at `generate-bundle` and then update the ci-index job. It's one of those chicken/egg problems.

Reply:
Good call. Done.